src/site/content/en/progressive-web-apps/add-manifest/index.md

```diff
@@ -4,23 +4,25 @@ title: Add a web app manifest
 authors:
   - petelepage
 date: 2018-11-05
-updated: 2020-02-18
+updated: 2020-02-24
 description: |
   The web app manifest is a simple JSON file that tells the browser about your
   web application and how it should behave when installed on the user's mobile
   device or desktop.
+tags:
+  - post
```

Comment:
Adding this will make the document appear on the blog. Just confirming that you want that to happen?

Reply:
If I don't add that, when a user clicks on the "Progressive Web App" tag that is added to the bottom of each article ([example](https://deploy-preview-2200--web-dev-staging.netlify.com/what-are-pwas/)), the article isn't listed there.

It's not *exactly* what I want, but it also feels weird if the user clicks on the tag, goes to the [tag page](https://deploy-preview-2200--web-dev-staging.netlify.com/tags/progressive-web-apps/) and doesn't see the article that they're expecting to see.

Reply:
ah yes, that is a bug that I believe @MichaelSolati was planning to fix. Maybe we can get that resolved today.

Reply:
Fixed and deployed minutes ago

Reply:
Awesome - thanks. Have removed the `tag: post` from pages.